_ml-commons-plugin/cluster-settings.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: ML Commons cluster settings
+has_children: false
+nav_order: 10
+---
+
+# ML Commons cluster settings
+
+This page provides an overview of `opensearch.yml` settings that can be configured for the ML commons plugin.
+
+| **Setting Name** | **Default Value** | **Value Range** | **Description** |
+|:---:|:---:|:---:|:---:|
+| `plugins.ml_commons.only_run_on_ml_node` | false | true or false | If `true`, ML Commons task and models run machine learning (ML) tasks on ML nodes only. If `false`, tasks and models run on ML nodes first. If no ML nodes exist, tasks and model run on data nodes. Don't set as "false" on production cluster. |
+| `plugins.ml_commons.task_dispatch_policy` | round_robin |  round_robin or least_load | Default is `round_robin`. `round_  robin` dispatches ML tasks to ML nodes using round robin routing. `least_load` gathers all an ML nodes' runtime information, like JVM heap memory usage and running tasks, then dispatches tasks to the ML node with the least load. |
+| `plugins.ml_commons.sync_up_job_interval_in_seconds` | 10 | From 0 (included) to 86400 (included) | If set as 0, ML Commons immediately stops any syncup jobs |
```

Comment:
The ranges are awkward to read. Is there a reason it isn't like:
```
Min: 0 | Max: 86400
```

Or even simpler, just:
```
0-86400
```

Technically, the representation for an inclusive range would be written like:
```
[0,86400]
```

If there's a desire to make it verbose, what about rewriting these ranges like:
```
Integer [0,86400]
```

Also, 100.233548 is between 0 and 86400. As a user, can I specify that? Or does it need to be an integer? The doc doesn't actually tell me I can't use a floating point value.

Reply:
I've used `Valid values are integers in the [0, 86400] range.` Brackets are universally understood as an inclusive range, but if we'd like to be explicit, we can add `, inclusive.`